src/pkg/storage/storage.go

```diff
@@ -168,7 +169,16 @@ func mustMatchConfig(
 		}
 
 		vv := cast.ToString(g.Get(tomlK))
-		if v != vv {
+
+		areEqual := false
+
+		if path.IsValidPath(v) && path.IsValidPath(vv) {
+			areEqual = path.ArePathsEquivalent(v, vv)
+		} else {
+			areEqual = v == vv
+		}
```

Comment:
It should be possible to skip the valid check and call `path.ArePathsEquivalent` for any pair of values, right?  Whether or not they clear os.Stat doesn't actually change how ArePathsEquivalent compares them.

Reply:
The value pairs can be any string other than paths. So the idea was to compare paths only if they qualify as paths.